resources/store/actions.js

```diff
@@ -145,8 +145,11 @@ module.exports = {
 			reviewBatch,
 			reviewImageLabelsRequest;
 
-		// Set the review state for all tags which could be displayed to the user
-		reviewBatch = displayedTags.map( function ( tag ) {
+		// Set the review state for non-user-provided tags which could be
+		// displayed to the user
+		reviewBatch = displayedTags.filter( function ( tag ) {
```

Comment:
I *assume* it's okay to send a review image labels request that has an empty review batch (for cases when the user could not see a single provided label and only confirmed their own custom ones)...

Reply:
The non-displayed tags are being added to the batch though, right?

Reply:
Right, see line 158 below. In that case there's no way an empty batch can be sent.